content/en/docs/setup/install/multicluster/multi-primary_multi-network/index.md

```diff
@@ -0,0 +1,314 @@
+---
+title: Install Multi-Primary on Different Networks
+description: Install an Istio mesh across multiple primary clusters on different networks.
+weight: 30
+icon: setup
+aliases:
+    - /docs/setup/kubernetes/multicluster-install/
+    - /docs/setup/kubernetes/multicluster/
+    - /docs/setup/kubernetes/install/multicluster/
+keywords: [kubernetes,multicluster]
+test: yes
+owner: istio/wg-environments-maintainers
+---
+Follow this guide to install the Istio control plane on both `cluster1` and
+`cluster2`, making each a {{< gloss >}}primary cluster{{< /gloss >}}. Cluster
+`cluster1` is on the `network1` network, while `cluster2` is on the
+`network2` network. This means there is no direct connectivity between pods
+across cluster boundaries.
+
+Before proceeding, be sure to complete the steps under
+[Before you begin](/docs/setup/install/multicluster#before-you-begin).
+
+In this configuration, both `cluster1` and `cluster2` observe the API Servers
+in each cluster for endpoints.
+
+Service workloads across cluster boundaries communicate indirectly, via
+dedicated gateways for [east-west](https://en.wikipedia.org/wiki/East-west_traffic)
+traffic. The gateway in each cluster must be reachable from the other cluster.
+
+{{< image width="75%"
+    link="arch.svg"
+    caption="Multiple primary clusters on separate networks"
+    >}}
+
+## Configure `cluster1` as a primary
+
+Create the Istio configuration for `cluster1`:
+
+{{< text bash >}}
+$ cat <<EOF > cluster1.yaml
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  values:
+    global:
+      meshID: mesh1
+      multiCluster:
+        clusterName: cluster1
+      network: network1
+EOF
+{{< /text >}}
+
+Apply the configuration to `cluster1`:
+
+{{< text bash >}}
+$ istioctl install --context="${CTX_CLUSTER1}" -f cluster1.yaml
+{{< /text >}}
+
+## Install the east-west gateway in `cluster1`
+
+Install a gateway in `cluster1` that is dedicated to
+[east-west](https://en.wikipedia.org/wiki/East-west_traffic) traffic. By
+default, this gateway will be public on the Internet. Production systems may
+require additional access restrictions (e.g. via firewall rules) to prevent
+external attacks. Check with your cloud vendor to see what options are
+available.
+
+{{< text bash >}}
+$ MESH=mesh1 CLUSTER=cluster1 NETWORK=network1 \
+    @samples/multicluster/gen-eastwest-gateway.sh@ | \
```

Comment:
Super nit for later: Can we change this shell script to take its arguments as arguments instead of env vars? On first glance this looked bizarre because why am I running a script to generate something if it takes no arguments, why not just put the generated file in samples. Then I realized arguments were passed as env vars. Ugh.

Reply:
Also we should rename it to "expansion-gateway rather than eastwest-gateway. Since we use it in some scenarios for control plane traffic only.

Reply:
I'm not sure expansion is a better term. East-west at least a widely-used term that implies the general direction of traffic. Can you explain why `expansion` is better here?

Reply:
After giving it more thought, I'm warming up to the name. I raised https://github.com/istio/istio/issues/28196 for tracking ... we'll probably change in the first patch release.

Reply:
Also raised https://github.com/istio/istio/issues/28197 for changing the script to use arguments.